### ISSUE TITLE
Content area input box pastes HTML styling from source

### ISSUE THREAD

**Issue by sk209** [soflyy/oxygen-bugs-and-features#2241]
**Describe the bug**
The newly added content box for text elements does not remove HTML styling from the source. Pasting any pre styled text (e.g.. From another website or document) will carry over the styling into the Oxygen builder, which is not the expected behaviour.

**Steps required to reproduce the bug: **
1. Add a text element in Oxygen builder
2. Copy any styled text from another website
3. In Oxygen builder, paste the copied text into the content box on the left sidebar.
4. The pasted text will carry over styling from the original website. The styling is shown as inline HTML in the frontend.

**Comment by pkboy** [comment]
If you paste without formatting, Option+Shift+Command+V on macOS, Control+Shift+V on Windows, then text will be pasted without formatting.

Formatting carrying over from the source text when pasting has been expected behaviour for a while. For example, copy styled text from a site and paste into a word processor.

**Comment by sk209** [comment]
You're absolutely right, but the primary purpose of this edit box is to edit text only, not apply styling since that is done in the builder via CSS.

For e.g. If I let my clients use this text box to edit content, they might paste in pre-formatted text which will override the website styles.

**Comment by pkboy** [comment]
> For e.g. If I let my clients use this text box to edit content, they might paste in pre-formatted text which will override the website styles.

This makes sense.
Guess it comes down to whether this behaviour (keeping formatting) is intended by the developers since Rich Text exists.

**Comment by YammYcoding** [comment]
I also see this issue here. Then it caused another side effect. After pasting the text copied from a site, the font size stopped working, any size I typed would increase the line height but not the font size in the editor. happened in more than one occasion.  If pasting the text as html was intended, I suppose it breaking the font size wasn't expected.

Also. now this content prevents oxy toolbox content text from showing. There, at least, I could paste any text and it would not bring the source style with it, and plus I could see the html source along with the text. I'd rather have two content fields, one for the 'source' and another for the 'rendered' text.

**Comment by sk209** [comment]
Fixed in Oxygen 3.9 Beta 1
